packages/melos/lib/src/package.dart

```diff
@@ -379,6 +396,7 @@ class PackageFilters {
     String? diff,
     bool? includeDependencies,
     bool? includeDependents,
+    List<Glob>? category,
```

Comment:
Follow the same convention as the rest of the fields here and use the same name as in the constructor